fs_mark/fs_mark.py

```diff
@@ -0,0 +1,214 @@
+import trunner
+import time
+import os
+import yaml
+import re
+import pexpect
+import timing_data as t_data
+import psh.tools.psh as psh
+
+from datetime import datetime
+from trunner.ctx import TestContext
+from trunner.dut import Dut
+from trunner.types import TestResult, TestSubResult, Status
+
+# Targets with dummyfs
+dummy_list = ['armv7m7-imxrt105x-evk', 'armv7m7-imxrt106x-evk',
+              'armv7m7-imxrt117x-evk', 'armv7m4-stm32l4x6-nucleo']
+
+
+def clean(dut, test_name, ctx: TestContext) -> int:
+    global dirs
+
+    if ctx.target.name in dummy_list:
```

Comment:
so, as I mentioned above, I think this should be enough
```suggestion
    if not ctx.target.rootfs:
```